vcpkg/consume/binary-caching.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Reduce package build times with binary caching
+description: Learn to set up a binary cache to reduce rebuild times.
+author: vicroms
+ms.author: viromer
+ms.date: 8/13/2023
+ms.prod: vcpkg
+---
+# Reduce package build times with binary caching
+
+Most ports in the [vcpkg public registry](<https://github.com/Microsoft/vcpkg>) are built from
+source. Building from source ensures maximum compatiblity by using the same environment, build
+tools, and configurations you use for your project in your dependencies. However, this  can also
+result in duplicated builds when multiple projects, developers, or machines require the same
+dependencies.
+
+Setting up a binary cache allows vcpkg to save the build output of a package in a shared location
+and reuse it when asked to build the same package with the same configuration, saving time on
+rebuilds. 
+
+Binary caches can be hosted in a variety of environments, the most basic form of binary caching is a
+directory containing zipped archives of a package's build output. More advanced scenarios include
+NuGet package feeds hosted in [GitHub Packages](<https://docs.github.com/en/packages>), [Azure
+DevOps Artifacts](<https://learn.microsoft.com/azure/devops/artifacts/start-using-azure-artifacts?view=azure-devops&tabs=nuget>),
+or other such services.
+
+Binary caching is especially effective in scenarios where dependencies and configuration
+don't change often, such as in CI or build servers. Binary packages are particularly susceptible to
```

Comment:
"Binary caching is especially effective in scenarios where dependencies and configuration don't change often, such as in CI or build servers." - Consider specifying the abbreviation "CI" (Continuous Integration) upon first use.

"Binary packages are particularly susceptible to configuration changes" - Specify what "binary packages" are referring to.